modules/templates/mailer.go

```diff
@@ -44,11 +44,17 @@ func buildSubjectBodyTemplate(stpl *texttmpl.Template, btpl *template.Template,
 	}
 	if _, err := stpl.New(name).
 		Parse(string(subjectContent)); err != nil {
-		log.Warn("Failed to parse template [%s/subject]: %v", name, err)
+		log.Error("Failed to parse template [%s/subject]: %v", name, err)
+		if !setting.IsProd {
+			log.Fatal("Please fix the mail template error")
+		}
 	}
 	if _, err := btpl.New(name).
 		Parse(string(bodyContent)); err != nil {
-		log.Warn("Failed to parse template [%s/body]: %v", name, err)
+		log.Error("Failed to parse template [%s/body]: %v", name, err)
+		if !setting.IsProd {
+			log.Fatal("Please fix the mail template error")
+		}
```

Comment:
I'm of the opinion that behaviour differences between RUN_MODE should be kept minimal. Is it warranted here?

Reply:
Actually I think it should also "Fatal" (just like web page HTML templates)

But the old code just reports the error message and continues. So I chose to keep the old behavior at the moment.

So I think it is warranted that "the behavior is the same as the old code and the change is minimal", but it doesn't warrant that "the behavior is the same for production/dev"

Feel free to change the behavior.